scripts/test-math.ts

```diff
@@ -0,0 +1,53 @@
+import { Network } from "@invariant-labs/sdk-eclipse";
+import { createSnapshotForNetwork } from "../src/snap-points";
+import { IPointsJson } from "../src/types";
+import * as fs from "fs";
+import path from "path";
+import { BN } from "@coral-xyz/anchor";
+import { POINTS_PER_SECOND } from "../src/math";
+
+const testMath = async () => {
+  const firstTimestamp = await createSnapshotForNetwork(Network.TEST);
+  const previousPointsFirst: Record<string, IPointsJson> = JSON.parse(
```

Comment:
Instead of naming first and second it will be better to refer to it as distributed and expected